x-pack/plugins/aiops/common/constants.ts

```diff
@@ -19,3 +19,11 @@ export const RANDOM_SAMPLER_SEED = 3867412;
 export const CASES_ATTACHMENT_CHANGE_POINT_CHART = 'aiopsChangePointChart';
 
 export const EMBEDDABLE_CHANGE_POINT_CHART_TYPE = 'aiopsChangePointChart' as const;
+
+export const AIOPS_TELEMETRY_ID = {
+  LOG_RATE_ANALYSIS_RUN: 'aiops_log_rate_analysis_run',
+  LOG_PATTERN_ANALYSIS_RUN: 'aiops_log_pattern_analysis_run',
+  CHANGE_POINT_DETECTION_RUN: 'aiops_change_point_detection_run',
+  AIOPS_DEFAULT_SOURCE: 'ml_aiops_labs',
+  AIOPS_ANALYSIS_RUN_ORIGIN: 'aiops-analysis-run-origin',
+} as const;
```

Comment:
Sorry for the continued nitpicking 😅  — looking at the attributes now they serve different purposes so we probably shouldn't combine them all into one enum. Suggest to break out the `run` ones. 

If I read the code right `AIOPS_ANALYSIS_RUN_ORIGIN` is now the header name? Suggest to prefix with `kbn-` then. We now also have a mix of `source` and `origin` in the naming. Related to that, if you align this, maybe we can also make the prop name `source` more specific, what do you think?

Reply:
The header name can't begin with 'kbn' - I got an error. But good call on renaming the prop to match up.

Reply:
Renamed prop name to 'embeddingOrigin' in c973e760a6ad0fc945cbe11b91ef798903aca1b5